acasclient/lsthing.py

```diff
@@ -584,16 +584,45 @@ class BlobValue(object):
     These files must be small (< 1 GB) and will be stored in a `bytea` database column.
     """
 
-    def __init__(self, value=None, comments=None):
+    def __init__(self, value=None, comments=None, id=None, ls_value=None):
         """Create a BlobValue
 
         :param value: Bytes of file content, defaults to None
         :type value: bytes, optional
         :param comments: Filename as a string, defaults to None
         :type comments: str, optional
+        :param id: id as an int, defaults to None
+        :type id: int, optional
+        :param ls_value: ls_value as a <acasclient.lsthing.LsThingValue>, defaults to None
+        :type ls_value: <acasclient.lsthing.LsThingValue>, optional
         """
+        if ls_value is not None:
+            value = ls_value.blob_value
+            comments = ls_value.comments
+            id = ls_value.id
+        else:
+            if value is not None:
+                if isinstance(value, Path) or isinstance(value, str):
```

Comment:
It feels "easter-eggy" to overload the "value" field this way. I'd prefer this as a distinct argument, maybe `filepath` that will set `value` to the bytes and set `comments` to the file name